pkg/abstract/model/endpoint_rotator_config_test.go

```diff
@@ -206,8 +207,6 @@ func offsetDateTestHours(t *testing.T) {
 
 func offsetDateTestDays(t *testing.T) {
 	t.Parallel()
-	_ = os.Setenv("TZ", "Europe/Moscow") // this test is timezone aware
```

Comment:
this is a part of test, without it - it would become flacky and will fail in 00-00 UTC.